### DISCUSSION TITLE
use fossil instead of git for updating buckets

### DISCUSSION THREAD

**Discussion by goyalyashpal** [ScoopInstaller/Scoop#5886] [Help and Questions] [answered]
continuing from here: https://github.com/ScoopInstaller/Scoop/discussions/5885

so, on the client's win 7 system, in `extras/doublecmd` i navigated to the scoop-apps directory, and the `git-portable` or whatever was taking at the top consuming whopping 300 MB of space.

this is whopping due to following reasons:
* as that system is old and constrained on disk space.
* for non-tech people who do not use git, this is way too much of a consumption by a needless superfluous application

so, i'd like to explore using `main/fossil` vcs for pulling the git buckets. it was only 30-40 MB the last time i used that.
and it says that it can work with git. but i don't know if it can work with remote-gits too...

### versions

```console
$ scoop --version | head -n2
Current Scoop version:
v0.3.1 - Released at 2022-11-15

```

**Comment by rashil2000** [comment, marked as answer]
See also https://github.com/ScoopInstaller/Scoop/issues/4881
